src/script/components/InputBar/hooks/useResizeTarget.ts

```diff
@@ -19,19 +19,34 @@
 
 import {useEffect, DependencyList} from 'react';
 
-const useResizeTarget = (element: HTMLElement | null, targetElement: HTMLElement | null, deps?: DependencyList) => {
+const alignScrollBars = (shadowInput: HTMLElement, textarea: HTMLElement) => {
+  const shouldForceScrollbar = textarea.clientHeight < textarea.scrollHeight;
+
+  shadowInput.style.overflowY = shouldForceScrollbar ? 'scroll' : 'hidden';
+  textarea.style.overflowY = shouldForceScrollbar ? 'scroll' : 'hidden';
```

Comment:
According to [documentation](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollHeight) `scrollHeight` always shows the height of an element's content, including content not visible on the screen due to overflow. Using `scroll` and `hidden` values, we're making sure that scroll behaves the same way on both elements (`overflow: auto` behaviour differs on `textarea` and `div`).

Reply:
Please add this as a comment in the code as well, that would help for future understanding :)

Reply:
added 👌